### ISSUE TITLE
Two or more same autolinks markdown string combined to a single anchor element when given without space

### ISSUE THREAD

**Issue by vivinkrishna-ni** [ni/nimble#1714]
# 🐛 Bug Report

In rich text editor/viewer, prosemirror-markdown parser is used to convert a markdown string to a DOM. Here if we provide two or more same autolinks as a markdown input to the components they convert it as a single anchor element. This is also an issue in the `@mention` feature if the user mentions two or more users without a space in between they convert it to a single user.

Created issue in prosemirror-markdown: https://github.com/ProseMirror/prosemirror-markdown/issues/115

## 💻 Repro or Code Sample

Add two same autolink markdown strings/mention format to the viewer input. For example, `<user:1><user:1>`.
![image](https://github.com/ni/nimble/assets/123377523/c36118ce-6caa-43d9-b715-331ed4f0fce3)
 

## 🤔 Expected Behavior

If it is a mention element, it should be rendered as two same mentions like below.
![image](https://github.com/ni/nimble/assets/123377523/817cbcfd-10c7-4f1a-a4b8-b00d95ae175f)

If it is a link, it should be rendered as two same links like below.
![image](https://github.com/ni/nimble/assets/123377523/d9ddf2c3-d2a7-45d3-a329-42605489463a)

## 😯 Current Behavior

For mention and link:
![image](https://github.com/ni/nimble/assets/123377523/ff9bba53-ead9-4e15-8ecd-3acb363e3949)

## 🔦 Context

[Bug 2605451](https://dev.azure.com/ni/DevCentral/_workitems/edit/2605451): Same username tagged without spaces in between not rendering as expected when editing

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

**Comment by fredvisser** [comment]
Duplicate AzDO work item – [Bug 2645465](https://dev.azure.com/ni/DevCentral/_workitems/edit/2645465): [@mention] Two or more same autolinks markdown string combined to a single anchor element when given without space

**Comment by fredvisser** [comment]
The issue is blocked on an upstream issue (https://github.com/ProseMirror/prosemirror-markdown/issues/115).

Let's review this quarterly along with other low sev `blocked` bugs.

**Comment by m-akinc** [comment]
ProseMirror closed their bug as something they will not fix. Given this only affects links/mentions where the _same_ URL/user is linked/mentioned in succession, without an intervening space, it seems quite low severity. I'm closing as "won't fix" due to low ROI. FYI @fredvisser

**Comment by jattasNI** [comment]
The linked AzDO issue was also closed as low ROI, so I agree closing this makes sense.
